docs/src/content/docs/fr/guides/i18n.mdx

```diff
@@ -179,35 +187,7 @@ Vous pouvez fournir des traductions pour les langues supplémentaires que vous s
 
    Voici les valeurs anglaises par défaut des chaînes de caractères existantes avec lesquelles Starlight est livré :
 
-   ```json
-   {
-   	"skipLink.label": "Skip to content",
-   	"search.label": "Search",
-   	"search.shortcutLabel": "(Press / to Search)",
-   	"search.cancelLabel": "Cancel",
-   	"search.devWarning": "Search is only available in production builds. \nTry building and previewing the site to test it out locally.",
-   	"themeSelect.accessibleLabel": "Select theme",
-   	"themeSelect.dark": "Dark",
-   	"themeSelect.light": "Light",
-   	"themeSelect.auto": "Auto",
-   	"languageSelect.accessibleLabel": "Select language",
-   	"menuButton.accessibleLabel": "Menu",
-   	"sidebarNav.accessibleLabel": "Main",
-   	"tableOfContents.onThisPage": "On this page",
-   	"tableOfContents.overview": "Overview",
-   	"i18n.untranslatedContent": "This content is not available in your language yet.",
-   	"page.editLink": "Edit page",
-   	"page.lastUpdated": "Last updated:",
-   	"page.previousLink": "Previous",
-   	"page.nextLink": "Next",
-   	"404.text": "Page not found. Check the URL or try using the search bar.",
-   	"aside.note": "Note",
-   	"aside.tip": "Tip",
-   	"aside.caution": "Caution",
-   	"aside.danger": "Danger",
-   	"fileTree.directory": "Directory"
-   }
-   ```
+	 <UIStringsList />
```

Comment:
```suggestion
   <UIStringsList />
```

When Prettier will run, this would be formatted as:

```mdx
   {' '}
   <UIStringsList />
```

This is something we do not want. Is your editor/IDE not picking the Prettier configuration in the repo or you're using a different one maybe?

Reply:
> When Prettier will run, this would be formatted as:
> 
> ```mdx
>    {' '}
>    <UIStringsList />
> ```
> 
> This is something we do not want. Is your editor/IDE not picking the Prettier configuration in the repo or you're using a different one maybe?

I think yes, I'm gonna configurate again my IDE, I reinstalled W11 last week